scripts/Tools/clean_build

```diff
@@ -86,7 +86,13 @@ if( $cnt==0){
     $opts{glc} = 1;
     $opts{ice} = 1;
     $opts{rof} = 1;
-    # Do not clean shared land library in a test case unless explicitly requested
```

Comment:
I'm not at all tied to my particular comment - I just felt that some comment was needed here explaining why lnd is treated differently. If you think of a better way to explain this, I'm happy to go with that.

It seems like your comment is saying something similar to what I have in parentheses. Should I just move what's in parentheses to the start of this comment?